cipherstash-pg.gemspec

```diff
@@ -23,12 +23,14 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features)/}) }
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features|vendor)/}) }
   end
   spec.extensions    = ["ext/extconf.rb"]
   spec.require_paths = ["lib"]
   spec.cert_chain    = ["certs/ged.pem"]
   spec.rdoc_options  = ["--main", "README.rdoc",
                         "--title", "PG: The Ruby PostgreSQL Driver"]
   spec.extra_rdoc_files = `git ls-files -z *.rdoc lib/*.rb lib/*/*.rb ext/*.c ext/*.h`.split("\x0")
+  spec.files << "vendor/driver/database-extensions/postgresql/install.sql"
+  spec.files << "vendor/driver/database-extensions/postgresql/uninstall.sql"
```

Comment:
This path isn't valid; there's nothing in `vendor/driver`.

(The submodule isn't present.)

Reply:
@damncabbage oh, hmm, it's present when I view it locally? 

<img width="734" alt="Screenshot 2023-02-21 at 2 01 24 pm" src="https://user-images.githubusercontent.com/26052576/220236783-5eae29b1-4afe-447f-895f-74e7c44c84d4.png">

Reply:
@damncabbage I've fixed that up, the submodule has been added in the PR.